maintenance/migratestatus/migratestatus.go

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/golang/glog"
+	"github.com/google/go-github/github"
+	"k8s.io/test-infra/maintenance/migratestatus/migrator"
+)
+
+func main() {
+	var tokenFile, org, repo, modeFlag, origContext, newContext string
+	var dryRun bool
+
+	flag.StringVar(&tokenFile, "tokenfile", "", "The file containing the token to use for authentication.")
+	flag.StringVar(&org, "org", "", "The organization that owns the repo.")
+	flag.StringVar(&repo, "repo", "", "The repo needing status migration.")
+	flag.BoolVar(&dryRun, "dry-run", true, "In dry run mode, no modifying actions will be taken.")
+
+	flag.StringVar(&modeFlag, "mode", "", "The mode to run. Possible values are [deprecate|duplicate|replace].")
```

Comment:
done. I changed the names of the modes to copy, retire, and move (copy and retire).